src/app/profile/profile.page.html

```diff
@@ -7,13 +7,18 @@
 <ion-content fullscreen>
     <ion-list *ngIf="(info$ |async) as info">
       <ion-item class="default-avatar">
+        <input type="file" name="file" id="file" class="file-input" (change)="picChange($event)"
+      accept="image/gif, image/jpeg, image/png" />
+        <label for="file">
           <ion-avatar>
-            <img #profilePic  [src]="getAvatar(info.photoURL)">
+            <img #profilePic [src]="(info.photoURL)?info.photoURL:avatarSrc">
+            <img class="plus-icon" src="assets/icon/plus.svg" *ngIf="!isProfilePicSelected">
```

Comment:
Plus icon only displays when there is no profile pic selected. Currently it's coming for all the scenarios.